tests/src/test/scala/whisk/core/entity/test/SchemaTests.scala

```diff
@@ -70,6 +69,46 @@ class SchemaTests extends FlatSpec with BeforeAndAfter with ExecHelpers with Mat
     a[DeserializationException] should be thrownBy Privilege.serdes.read("???".toJson)
   }
 
+  behavior of "TransactionId"
+
+  it should "serdes a transaction id without extraLogging parameter" in {
+    val txId = 4711
+    val txWithoutExtraLogging = TransactionId(txId)
+
+    // test serialization
+    val serializedTxIdWithoutParameter = TransactionId.serdes.write(txWithoutExtraLogging)
+    serializedTxIdWithoutParameter match {
+      case JsArray(Vector(JsNumber(id), JsNumber(_))) =>
+        assert(id == txId)
+      case JsArray(Vector(JsNumber(_), JsNumber(_), JsBoolean(_))) =>
```

Comment:
`case _ => assert(false)` since anything else is not expected?

Reply:
I have now added that case